jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/bucketActions/OpenPrefixedBucketViewerAction.kt

```diff
@@ -6,20 +6,20 @@ package software.aws.toolkits.jetbrains.services.s3.bucketActions
 import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.project.DumbAware
 import com.intellij.openapi.ui.Messages
-import icons.AwsIcons
 import software.aws.toolkits.jetbrains.core.explorer.actions.SingleResourceNodeAction
 import software.aws.toolkits.jetbrains.services.s3.S3BucketNode
 import software.aws.toolkits.jetbrains.services.s3.openEditor
+import software.aws.toolkits.resources.message
 import software.aws.toolkits.telemetry.Result
 import software.aws.toolkits.telemetry.S3Telemetry
 
-class OpenPrefixedBucketViewerAction : SingleResourceNodeAction<S3BucketNode>("View Bucket with prefix..."), DumbAware {
+class OpenPrefixedBucketViewerAction : SingleResourceNodeAction<S3BucketNode>(), DumbAware {
     override fun actionPerformed(selected: S3BucketNode, e: AnActionEvent) {
         val prefix = Messages.showInputDialog(
             selected.nodeProject,
-            "dialog message",
-            "dialog title",
-            AwsIcons.Resources.S3_BUCKET
+            message("s3.open.viewer.prefix.message"),
```

Comment:
it'd be nice if there were a clean way to provide sample empty text

Reply:
Would be nice as an enhancement one day